components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -317,6 +330,7 @@ export default class CollectionItem extends mixins(ChainMixin, PrefixMixin) {
       this.$router.push({ name: 'errorcollection' })
       return
     }
+    this.firstMintDate = collectionEntity.createdAt
```

Comment:
where setter 👁️

Reply:
Do u mean where is the initialize? It's in the CreatedAtMixin

`export default class CreatedAtMixin extends Vue {
  protected firstMintDate = new Date()

  get formattedTimeToNow() {
    return this.firstMintDate
      ? formatDistanceToNow(new Date(this.firstMintDate), { addSuffix: true })
      : ''
  }
}
`